haste_storage_client/core.py

```diff
@@ -4,22 +4,39 @@
 
 
 class HasteStorageClient:
+    INTERESTINGNESS_DEFAULT = 1.0
+
     def __init__(self,
                  stream_id,
                  host,
                  port,
-                 keystone_auth):
+                 keystone_auth,
+                 interestingness_model=None,
+                 storage_policy=None,
+                 default_storage_class='swift'):
         """
         :param stream_id: String ID for the stream session - used to group all the data
         (unique for each execution of the experiment)
         :param host: Hostname/IP of database server.
         :param port: Database server port. Usually 27017.
         :param keystone_auth: OpenCloud keystone auth v3 password object,
         see: https://docs.openstack.org/keystoneauth/latest/api/keystoneauth1.identity.v3.html#module-keystoneauth1.identity.v3.password
+        :param interestingness_model: InterestingnessModel to determine interestingness of the document,
+        and hence the intended storage class.
+        :param storage_policy: policy mapping interestingness to storage class(es). Supported are 'swift' and
```

Comment:
from discussion this morning: can map single document to multiple storage classes depending on rules.